src/hist.jl

```diff
@@ -60,6 +60,44 @@ push!(Plots._histogram_like, :ea_histogram)
     ()
 end
 @shorthands testhist
+
+# ---------------------------------------------------------------------------
+# grouped histogram
+
+@userplot GroupedHist
+
+@recipe function f(p::GroupedHist)
+    @assert length(p.args) == 2
+    v = p.args[1]
+    gs = p.args[2]
```

Comment:
Doesn't something like `gs = get(plotattributes, :group, nothing)` work here, as in [GroupedBoxplot](https://github.com/JuliaPlots/StatsPlots.jl/blob/master/src/boxplot.jl#L147)? It'd be nicer to have a consistent interface.